gym/f110_gym/envs/f110_env.py

```diff
@@ -38,32 +38,36 @@
 # gl
 import pyglet
 
-pyglet.options['debug_gl'] = False
+pyglet.options["debug_gl"] = False
 from pyglet import gl
 
-# constants
-
 # rendering
 VIDEO_W = 600
 VIDEO_H = 400
 WINDOW_W = 1000
 WINDOW_H = 800
 
+# config map
+CONFIG_MAP = {
+    "rk4": Integrator.RK4,
+    "euler": Integrator.Euler,
+}
+
```

Comment:
Move the mapping string to Integrator in the `Integrator.from_string(integrator: str)` method

Reply:
```suggestion
```